src/aurora/2dafile.cpp

```diff
@@ -0,0 +1,705 @@
+/* xoreos-tools - Tools to help with xoreos development
```

Comment:
You need to replace the copyright note comment header in all the files you added, with the one for Phaethon.